gradio/documentation.py

```diff
@@ -16,18 +17,41 @@ def set_documentation_group(m):
         classes_to_document[m] = []
 
 
-def document(*fns):
+def extract_instance_attr_doc(cls, attr):
+    code = inspect.getsource(cls.__init__)
+    lines = [line.strip() for line in code.split("\n")]
+    found_attr = False
+    i = 0
+    for i, line in enumerate(lines):
+        if line.startswith("self." + attr):
+            found_attr = True
+            break
+    assert found_attr, f"Could not find {attr} in {cls.__name__}"
+    start_line = lines.index('"""', i)
+    assert start_line != -1, f"Could not find docstring for {attr} in {cls.__name__}"
+    end_line = lines.index('"""', start_line + 1)
+    assert (
+        end_line != -1
+    ), f"Could not find end of docstring for {attr} in {cls.__name__}"
+    doc_string = " ".join(lines[start_line + 1 : end_line])
+    return doc_string
+
+
+def document(*fns, inherit=False):
     """
     Defines the @document decorator which adds classes or functions to the Gradio
     documentation at www.gradio.app/docs.
 
     Usage examples:
     - Put @document() above a class to document the class and its constructor.
-    - Put @document(fn1, fn2) above a class to also document the class methods fn1 and fn2.
+    - Put @document("fn1", "fn2") above a class to also document methods fn1 and fn2.
+    - Put @document("*fn3") with an asterisk above a class to document the instance attribute methods f3.
```

Comment:
Could we change the syntax to `Class().fn_name` as it's more obvious to someone reading the codebase about what's happening? Still easy to parse (just look for a `.`)

Reply:
what do you mean? Can you give an example

Reply:
Instead of 

```py
@document("*change")
```

which seems kind of magical, do this instead:

```py
@document("Textbox().change")
```

Reply:
That does not distinguish between instance bound or class bound methods, as they both can be called in that manner. It's also weird to have to rewrite the class name twice like this, whats the point of the decorator. if you have to write the class name again

```python
@document("Editable.edit")
class Editable
  ...
```

Reply:
But not a big deal, ignore this